graftm/unpack_sequences.py

```diff
@@ -35,6 +37,10 @@ class UnexpectedFileFormatException(Exception): pass
     
     def __init__(self, read_file):
         self.read_file   = read_file
+        self.slash_ending_regex = re.compile('.*/[12]$')
```

Comment:
I don't think this should be an instance variable, as it is only used internally to this class. Instead, simply define it as a regular variable in `sequence_type`?